vm/src/vm.rs

```diff
@@ -42,6 +43,23 @@ use num_bigint::BigInt;
 #[cfg(feature = "rustpython-compiler")]
 use rustpython_compiler::{compile, error::CompileError};
 
+// Signal triggers
+// TODO: 64
+pub const NSIG: usize = 10;
+
+pub static mut TRIGGERS: [AtomicBool; NSIG] = [
```

Comment:
Having this variable as a static global would prevent multiple VirtualMachine instances. We can solve this, by moving the triggers struct into a RefCell. Place this RefCell into the signals.rs file. In the signals.rs file, create a global cell, with a trigger list. Each vm then can register a trigger list to.

Okay, my explanation is poor. Lets see how this evolves further :).